core/tests/protractor_utils/editor.js

```diff
@@ -1349,6 +1349,42 @@ var _runFromFeedbackTab = function(callbackFunction) {
   return result;
 };
 
+var getSuggestionThreads = function() {
+  return _runFromFeedbackTab(function() {
+    var suggestionRowClassName = '.protractor-test-oppia-feedback-tab-row';
+    var threads = [];
+
+    return element.all(by.css(suggestionRowClassName)).then(function(rows) {
+      rows.forEach(function() {
+        element(by.css('.protractor-test-exploration-feedback-subject'))
+          .getText().then(function(subject) {
+            threads.push(subject);
+          });
+      });
+      return threads;
+    });
+  });
+};
+
+var acceptSuggestion = function(suggestionDescription) {
+  return _runFromFeedbackTab(function() {
+    var suggestionRowClassName = '.protractor-test-oppia-feedback-tab-row';
+    element.all(by.css(suggestionRowClassName)).then(function(rows) {
+      var matchingSuggestionRows = rows.filter(function() {
+        return element(by.css('.protractor-test-exploration-feedback-subject')).
+          getText().then(function(subject) {
+            return suggestionDescription.indexOf(subject) !== -1;
```

Comment:
Just wondering, any reason to not check direct equality? May want to add an explanatory comment here explaining why.